packages/lerna/__tests__/index.test.ts

```diff
@@ -1,3 +1,19 @@
+/**
+ * Copyright 2022 Octorelease Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Should there be a license script (either with an eslint plugin or self-written) to avoid this from happening?

Reply:
The octorelease-lerna repo has the eslint license header plugin, but ESLint is only run on the "src" directory so test files were skipped. 🤦